js/plugins/code/base-text-element.js

```
@@ -1671,46 +1903,107 @@ class BaseTextElement extends HTMLElement {
+                            skipChildren = true;
                         }
                         break;
                     case 'blockquote':
-                        element = { elementName: 'quote-element', value: node.textContent.trim() };
+                        element = { elementName: 'quote-element', value: makeLinksClickable(node.innerHTML.trim()) };
+                        skipChildren = true;
                         break;
                     case 'pre':
                     case 'code':
-                        element = { elementName: 'code-element', value: node.textContent.trim() };
+                        const pre = node.tagName.toLowerCase() === 'pre' ? node : node.closest('pre');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)